website/src/roadmap/connectors-go-sdk.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: section
+title: "Connectors - Go SDK"
+permalink: /roadmap/connectors-go-sdk/
+section_menu: section-menu/roadmap.html
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+Roadmap for connectors developed using Go SDK.
+
+Go SDK plans to utilize currently available Java and Python connectors
+through cross-language transforms feature.
```

Comment:
Maybe add a link about `cross-language` to the multi sdk page.

Also, I think Go SDK might end up having SDF sources too.